packages/components/src/toggle-group-control/toggle-group-control-option-base/component.tsx

```diff
@@ -141,10 +141,14 @@ function ToggleGroupControlOptionBase(
 					<Ariakit.Radio
 						disabled={ disabled }
 						onFocusVisible={ () => {
+							const selectedValueIsNullish =
+								toggleGroupControlContext.value === null ||
+								toggleGroupControlContext.value === '';
```

Comment:
Due to how we add a layer of [`value` conversion](https://github.com/WordPress/gutenberg/blob/26e108deabcb6f58ffdf424f06887efb9f88333c/packages/components/src/toggle-group-control/toggle-group-control/as-radio-group.tsx#L51-L56) in controlled mode, the Ariakit store will hold a `null` value on first render with no value provided, but then an empty string after a consumer resets the `value` to `undefined`.